src/Tempest/Http/src/Routing/Matching/RouteMatch.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Routing\Matching;
+
+use Tempest\Http\Routing\Construction\MarkedRoute;
+
+final readonly class RouteMatch
+{
+    private function __construct(
+        public bool $isFound,
```

Comment:
I find it a bit weird that there's a boolean here. I'd expect this object to be always valid when a route was matched. In other words: it should be `null` when there's no match. Does that make sense?

Reply:
Personally I don't like to put a meaning onto the value "null" if I can help it. I can however, make "isFound" a method which checks on the existence of `$mark` field. That way we do still have an explicit meaning of a "not found" route. But you don't have the need for this explicit boolean  field. Would that suffice?

Reply:
Implemented isFound as computed, would that suffice?